src/_avatars.scss

```diff
@@ -22,21 +22,22 @@
 
 .avatar-large {
   @include avatar-base;
-  font-size: 1rem;
+  @include body;
```

Comment:
Good call! Btw you can combine these two lines like this: `@include body(var(--font-weight-bold));`

Reply:
Oooo